cmd/utils/flags.go

```diff
@@ -1310,12 +1315,85 @@ func SetPlsConfig(ctx *cli.Context, stack *node.Node, cfg *pls.Config) {
 
 	cfg.RootChainURL = ctx.GlobalString(PlasmaRootChainUrlFlag.Name)
 
-	if !ctx.GlobalIsSet(PlasmaRootChainContractFlag.Name) {
-		Fatalf("RootChain contract address must be set, using --rootchain.contract")
+	if ctx.GlobalIsSet(PlasmaRootChainContractFlag.Name) {
+		cfg.RootChainContract = common.HexToAddress(ctx.GlobalString(PlasmaRootChainContractFlag.Name))
 	}
-	cfg.RootChainContract = common.HexToAddress(ctx.GlobalString(PlasmaRootChainContractFlag.Name))
 
-	cfg.Genesis = core.DefaultGenesisBlock(cfg.RootChainContract)
+	switch {
+	case ctx.GlobalBool(TestnetFlag.Name):
+		if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 3
+		}
+		cfg.Genesis = core.DefaultTestnetGenesisBlock()
+	case ctx.GlobalBool(RinkebyFlag.Name):
+		if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 4
+		}
+		cfg.Genesis = core.DefaultRinkebyGenesisBlock()
+	case ctx.GlobalBool(DeveloperFlag.Name):
+		if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 1337
+		}
+		// hard coding for dev mode
+		var (
+			operatorKey, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
+			rootChainURL   = "ws://localhost:8546"
+
+			_statesRoot       [32]byte
+			_transactionsRoot [32]byte
+			_receiptsRoot     [32]byte
+
+			development      = true
+			NRELength        = big.NewInt(2)
+			statesRoot       = "0ded2f89db1e11454ba4ba90e31850587943ed4a412f2ddf422bd948eae8b164"
+			transactionsRoot = "56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"
+			receiptsRoot     = "0000000000000000000000000000000000000000000000000000000000000000"
+		)
+
+		t := time.NewTimer(2 * time.Second)
+		defer t.Stop()
+
+		rootchainBackend, err := ethclient.Dial(rootChainURL)
+		if err != nil {
+			Fatalf("Failed to connect rootchain: %v", err)
+		}
+
+		opt := bind.NewKeyedTransactor(operatorKey)
+		eh, _, _, err := epochhandler.DeployEpochHandler(opt, rootchainBackend)
+		if err != nil {
+			Fatalf("Failed to deploy epoch handler contract")
+		}
+
+		<-t.C
+		copy(_statesRoot[:], statesRoot)
+		copy(_transactionsRoot[:], transactionsRoot)
+		copy(_receiptsRoot[:], receiptsRoot)
+
+		rootchainContract, _, _, err := rootchain.DeployRootChain(opt, rootchainBackend, eh, development, NRELength, _statesRoot, _transactionsRoot, _receiptsRoot)
+		if err != nil {
+			Fatalf("Failed to deploy rootchain contract")
+		}
+		log.Info("Deploy rootchain contract", "address", rootchainContract)
+
+		account, err := ks.ImportECDSA(operatorKey, "")
+		if err != nil {
+			Fatalf("Faild to import operator account: %v", err)
+		}
+		log.Info("Unlocking operator account", "address", account.Address)
+
+		if err = ks.Unlock(account, ""); err != nil {
+			Fatalf("Failed to unlock operator account: %v", err)
+		}
+
+		cfg.Genesis = core.DeveloperGenesisBlock(uint64(ctx.GlobalInt(DeveloperPeriodFlag.Name)), rootchainContract, account.Address)
+		cfg.Operator = account
```

Comment:
`cfg.Operator` is set here.

https://github.com/Onther-Tech/plasma-evm/blob/e65b1a7ed0f0801febc9f9e23aa256c4c32ce3c2/cmd/utils/flags.go#L1285

Reply:
I think it should be run when you run `geth --dev`. So we need an operator account here.

Reply:
`geth --dev` only deploys RootChain contract before pls service is started. It doesn't means that operator account (`--rootchain.operator`) is fixed.